server/utils/hashPassword.js

```diff
@@ -0,0 +1,9 @@
+const bcrypt = require('bcryptjs');
+
+const hashPassword = (password) => bcrypt.hash(password, 10);
+
+const comparePassword = (password, hashedPassword, callback) => {
+  bcrypt.compare(password, hashedPassword, callback);
```

Comment:
compare can work with promises by omit the callback
check https://github.com/kelektiv/node.bcrypt.js#readme

Also as a personal opinion no need to have this file at all, just call `compare` and `sign` in the place you're using